lang/java/reef-io/src/test/java/org/apache/reef/io/network/group/impl/driver/CommunicationGroupDriverImplTest.java

```diff
@@ -79,11 +81,11 @@ public void onNext(final GroupCommunicationMessage msg) {
         .addReduce(ReduceOperatorName.class,
             ReduceOperatorSpec.newBuilder().setReceiverId(rootTaskId).build());
 
-    final ExecutorService pool = Executors.newFixedThreadPool(4);
-    final CountDownLatch countDownLatch = new CountDownLatch(4);
+    final ExecutorService pool = Executors.newFixedThreadPool(8);
+    final CountDownLatch countDownLatch = new CountDownLatch(8);
 
-    // first add child tasks and start them up
-    for (int index = 0; index < 3; index++) {
+    // first add two child tasks and start them up
+    for (int index = 0; index < 2; index++) {
```

Comment:
Is there any reason to change this value from 2 to 3? This might affect what this class was originally trying to test.